doc/sql.extensions/README.sql_security.txt

```diff
@@ -0,0 +1,153 @@
+SQL SECURITY.
+
+   Implements capability to run executable objects regarding SQL SECURITY clause.
+   SQL Standard (2003, 2011) Feature.
+
+Author:
+   Red Soft, roman.simakov(at)red-soft.ru
+
+Syntax is:
+
+CREATE TABLE <TABLENAME> (...) [SQL SECURITY {DEFINER | INVOKER}]
+ALTER TABLE <TABLENAME> ... [{ALTER SQL SECURITY {DEFINER | INVOKER} | DROP SQL SECURITY}]
+CREATE [OR ALTER] FUNCTION <FUNCTIONNAME> ... [SQL SECURITY {DEFINER | INVOKER}] AS ...
+CREATE [OR ALTER] PROCEDURE <PROCEDURENAME> ... [SQL SECURITY {DEFINER | INVOKER}] AS ...
+CREATE [OR ALTER] TRIGGER <TRIGGERNAME> ... [SQL SECURITY {DEFINER | INVOKER}] [AS ...]
+CREATE [OR ALTER] PACKAGE <PACKAGENAME> [SQL SECURITY {DEFINER | INVOKER}] AS ...
+
+Description:
+
+Makes it possible to execute some objects with permissions of either definer or invoker.
+By default INVOKER is used to keep backword compatibility.
+
+If INVOKER is specified a current set of privileges of the current user will ba used.
+If DEFINER - a set of privileges of object owner will be used to check an access to database objects used by this object.
+
+Trigger inherits SQL SECURITY option from TABLE but can overwrite it by explicit specifying.
```

Comment:
It would be logical to use the ALTER TRIGGER ... DROP SQL SECURITY. For tables ALTER TABLE ... DROP SQL SECURITY is no different from the ALTER TABLE ... ALTER SQL SECURITY INVOKER.